src/main/scala/symsim/ExactRL.scala

```diff
@@ -54,16 +54,18 @@ trait ExactRL[State, ObservableState, Action, Reward, Scheduler[_]]
   /** Execute a full learning episode (until the final state of agent is
     * reached).
     */
-  def learningEpisode(fR: (VF, List[Reward]), s_t: State): Scheduler[(VF, List[Reward])] =
+  def learningEpisode(fR: (VF, List[Reward], List[VF]), s_t: State): Scheduler[(VF, List[Reward], List[VF])] =
     def p(f: VF, r_acc: Reward, s: State, a: Action): Boolean = agent.isFinal(s)
```

Comment:
do you have any reason to keep the List[Reward] ?  This morning I thought that we could drop it.

Reply:
Yeah. I think keeping the code that we do not use around is problematic, especially that this function already has very complex type.  We move this to evaluation instead.